src/py42/clients/settings/device_settings.py

```diff
@@ -500,3 +508,15 @@ def pop(self, index=-1):
     def remove(self, value):
         self.data.remove(value)
         self.register_change()
+
+
+def _backup_set_is_legal_hold(bs):
```

Comment:
Could there ever be other reasons for this besides legal hold?

The criteria for the conditional seems generic, that is why I ask.

`_backup_set_has_locked_destinations()` ?

Reply:
Nope, for both the individual device and the org device defaults, it's only possible to lock individual destinations in the UI. Locking "destinations" as a parent category only happens for legal hold. 

It's maybe possible to craft custom json to set the destinations parent as locked?